flipperdevices/flipperzero-good-faps
# Fix FlipTDI and Mass Storage crashes

# What's new
  - Fixed #253 and #255
  - Fixed a bug similar to #253 in Mass Storage
  - Fixed the application name in an error message logged by FlipTDI

# Verification 

#253:
  - Connect the device to a USB charger
  - For both FlipTDI and Mass Storage:
    - Open the app
    - Verify that the device does not crash after ~3s
    - Exit the app
    - Verify that the device does not crash
  
#255:
  - Connect the device to a PC running a companion app
  - Launch FlipTDI
  - Verify that the device does not crash after ~3s
  - Exit the app
  - Verify that the device does not crash

# Checklist (For Reviewer)
